internal/cli/doctor.go

```diff
@@ -118,6 +119,20 @@ func (c *doctorCmd) run(ctx context.Context, args []string) error {
 	kVersion, err := wireK8sVersion(ctx)
 	printField("Version", kVersion, err)
 
+	kClient, err := wireK8sClient(ctx)
+	if err == nil {
+		// blackhole any warnings
+		newCtx := logger.WithLogger(ctx, logger.NewDeferredLogger(ctx))
+		registry := kClient.LocalRegistry(newCtx)
+		if registry.Empty() {
+			printField("Local Registry", "none", nil)
```

Comment:
not suuuper concerned about this, but it might confuse a user to see this if they'd, say, set `default_registry('localhost:5000')` in their Tiltfile. Consider calling the field "Auto-detected local registry"? But also, whatevs.

Reply:
changed it to Cluster Local Registry, which i think will end up being the term of art for this